apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/Internal/ExecuteHelper.cs

```diff
@@ -89,5 +90,55 @@ private static string GetResourceInfoTypeFromTrailers(RpcException exception)
                 return null;
             }
         }
+
+        /// <summary>
```

Comment:
This looks familiar - I know we've got the TCS version in GAX, but does this exist somewhere else?